extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.narayana.jta.runtime;
+
+import java.lang.annotation.*;
```

Comment:
Not really important, but I think we don't use use wildcard imports

Reply:
IntelliJ idea do this automatically but I can change it if needed

Reply:
Yes please.

Reply:
done